src/client/pages/EventPage/index.tsx

```diff
@@ -63,6 +64,7 @@ export type Props = RouteProps & ReturnType<typeof mapStateToProps> &
 
 interface EventPageState extends TabularPageState {
   teams: TESCTeam[];
+  err?: string;
```

Comment:
you should not need this - please remove. rely on `addEventDangerAlert` or such.